src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntity.php

```diff
@@ -26,6 +26,7 @@ class UniqueEntity extends Constraint
     public $message = 'This value is already used.';
     public $service = 'doctrine.orm.validator.unique';
     public $em = null;
+    public $repository = null;
```

Comment:
If I understand correctly, this is a class name of the entity related to the repository. So, `repository` is misleading here.